src/styles/card.scss

```diff
@@ -0,0 +1,117 @@
+@import './variables/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $light-gray;
+  border-radius: 5px;
+  margin: 16px;
+  height: auto;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+
+    .image {
+      background-image: url(../images/imac.jpeg);
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    line-height: 18px;
+    height: 36px;
+    width: 166px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $secondary-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    padding: 0 16px;
+    width: 100%;
+    justify-content: center;
```

Comment:
The `justify-content: center;` property is used here, but since the element is not a flex container, this property will have no effect. Consider removing it or ensuring the parent is a flex container.